nrf_modem/doc/gnss_interface.rst

```diff
@@ -397,15 +371,11 @@ Bits 10...15 are reserved and their value is ignored.
 
 By default, all QZSS PRNs (193...202) are enabled.
```

Comment:
PRNs 198 and 202 currently have no planned use and are there disabled by default in MFW2.0.0

Reply:
Updated as discussed.